examples/example-app.yaml

```diff
@@ -52,3 +53,7 @@ spec:
         - "/main"
         - "--process-metrics"
         - "--go-metrics"
+        resources:
+          requests:
+            cpu: 100m
```

Comment:
This is significantly higher than other applications we have. Is the example app really more resource intensive [than prometheus](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/manifests/operator.yaml#L630)?

Reply:
Not really, but not sure if it matters as long as it's not something ridiculous by default (500m)